packages/config/config/devices/0x0258/nas-wr01ze.json

```diff
@@ -191,14 +130,63 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disable Button",
+					"label": "Disable Report",
 					"value": 0
 				},
 				{
-					"label": "Enable Button",
+					"label": "Enable Report",
 					"value": 1
 				}
 			]
+		},
+		"7": {
+			"label": "Meter Report Interval",
+			"description": "Interval time reporting data to controller",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"8": {
+			"label": "Maxium over-load current",
+			"description": "maximum current the plug can provide to load",
+			"unit": "Ampere",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 16,
+			"defaultValue": 16,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"9": {
+			"label": "Power Report Threshold",
+			"description": "Power consumed change to be reported",
+			"unit": "cA",
```

Comment:
What kind of unit is this? I guess `"0.01 A"` which we prefer because it is obvious.